sql code/ksm-reunion-2025.sql

```diff
@@ -1038,12 +1085,26 @@ SELECT DISTINCT
   ,E.last_name   
   --- Attending Reunion 2024 - I will add this once registratio opens
   ,KR.CLASS_YEAR
+  ,case when KR.class_year = '1975' then '50th Milestone'
```

Comment:
Change to CFY rather than hard coding - otherwise you'll need to update/revise for future pulls